src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml

```diff
@@ -100,6 +99,36 @@
                 </span>
             </div>
         </div>
+        <div class="float-right">
+            @if (Model.Revision.Approvers.Contains(User.GetGitHubLogin()))
+            {
+                <form asp-resource="@Model.Review" class="form-inline float-left ml-2" asp-page-handler="RevertApproval" method="post" asp-requirement="@ApproverRequirement.Instance">

```

Comment:
Can we use a single toggle handler instead of two separate ones (see ToggleClosed)/